<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RPlot</class>
 <widget class="QMainWindow" name="RPlot">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPlainTextEdit" name="plainTextEdit">
    <property name="geometry">
     <rect>
      <x>633</x>
      <y>70</y>
      <width>161</width>
      <height>71</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>634</x>
      <y>40</y>
      <width>101</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Expression:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Plot_sin">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>310</y>
      <width>98</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Plot sin</string>
    </property>
   </widget>
   <widget class="GraphicsLayoutWidget" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>581</width>
      <height>501</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="Plot_cos">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>350</y>
      <width>98</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Plot cos</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Plot_2">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>390</y>
      <width>98</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Plot x^2</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_x_min">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>160</y>
      <width>101</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>x_min:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_x_max">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>210</y>
      <width>101</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>x_max:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_pts">
    <property name="geometry">
     <rect>
      <x>638</x>
      <y>250</y>
      <width>101</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>no of pts:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_x_min">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>155</y>
      <width>113</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_x_max">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>205</y>
      <width>113</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_pts">
    <property name="geometry">
     <rect>
      <x>708</x>
      <y>244</y>
      <width>71</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GraphicsLayoutWidget</class>
   <extends>QGraphicsView</extends>
   <header>pyqtgraph</header>
   <slots>
    <slot>sin_pressed()</slot>
    <slot>cos_pressed()</slot>
    <slot>square_pressed()</slot>
   </slots>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>Plot_sin</sender>
   <signal>clicked()</signal>
   <receiver>graphicsView</receiver>
   <slot>sin_pressed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>706</x>
     <y>361</y>
    </hint>
    <hint type="destinationlabel">
     <x>420</x>
     <y>243</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Plot_cos</sender>
   <signal>clicked()</signal>
   <receiver>graphicsView</receiver>
   <slot>cos_pressed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>679</x>
     <y>401</y>
    </hint>
    <hint type="destinationlabel">
     <x>514</x>
     <y>309</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Plot_2</sender>
   <signal>clicked()</signal>
   <receiver>graphicsView</receiver>
   <slot>square_pressed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>692</x>
     <y>441</y>
    </hint>
    <hint type="destinationlabel">
     <x>411</x>
     <y>362</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
